### PR TITLE
Update AUTHORS.txt

### PR DESCRIPTION
**Pull request by robertfoss**

### CODE REVIEW

File: AUTHORS.txt [hunk 1]
```diff
@@ -709,6 +709,7 @@ Rob Hoelz <rob@hoelz.ro>
 Robert Buonpastore <robert.buonpastore@gmail.com>
 Robert Clipsham <robert@octarineparrot.com>
 Robert Gawdzik <rgawdzik@hotmail.com>
+RObert Foss <dev@robertfoss.se>
```
Comment (Manishearth) [inline AUTHORS.txt]: Nit: Accidental capital O?

**Comment by alexcrichton** [comment]
r? @brson 

Also cc #

**Comment by alexcrichton** [comment]
er, cc #25196

**Comment by brson** [comment]
Skipped straight to beta-accepted.

**Pull request merged**